frontend/src/components/Modal/AddBidModal/AddBidModal.tsx

```diff
@@ -0,0 +1,52 @@
+import style from "./addBidModal.module.scss";
+import { Button } from "components";
+import { useAppSelector } from "store/store";
+import React, { useState } from "react";
+import { BidOffer } from "interfaces";
+
+enum buttonTypes {
```

Comment:
suggestion: move this into a common enum, since it's defined and used in another component as well `(frontend/src/components/button/Button.tsx)`